### DISCUSSION TITLE
Language support

### DISCUSSION THREAD

**Discussion by ManindraDeMel** [crowdin/translate-readme#21] [Q&A] [answered]
I'm Just wondering what languages this action supports. As its not clear in the readme

**Comment by andrii-bodnar** [comment, marked as answer]
Hi @ManindraDeMel, it supports any language.

**Reply by ManindraDeMel** [reply]
how do i specify what languages to convert to (like from english -> french, Spanish, mandarin?

**Reply by andrii-bodnar** [reply]
@ManindraDeMel it should be specified within your Crowdin project

**Reply by ManindraDeMel** [reply]
Thanks
